src/main/java/com/checkmarx/flow/service/FlowService.java

```diff
@@ -21,17 +22,26 @@
     @Async("webHook")
     public void initiateAutomation(ScanRequest scanRequest) {
         ScanResults combinedResults = new ScanResults();
+        boolean isAtLeastOneScannerIsEnabled = false;
 
         String effectiveProjectName = projectNameGenerator.determineProjectName(scanRequest);
         scanRequest.setProject(effectiveProjectName);
 
         for (VulnerabilityScanner currentScanner : scanners) {
             if (currentScanner.isEnabled()) {
+                isAtLeastOneScannerIsEnabled = true;
                 ScanResults scanResults = currentScanner.scan(scanRequest);
                 combinedResults.mergeWith(scanResults);
             }
         }
-
+        if (!isAtLeastOneScannerIsEnabled) {
+            handleNoScannerIsEnabled();
+        }
         resultsService.publishCombinedResults(scanRequest, combinedResults);
     }
+
+    private void handleNoScannerIsEnabled() {
+        String errorMessage = "The defined scanners are not supported. Please make sure you're using one of the following: [sast, sca]";
```

Comment:
- where this exception being caught?
  why not write the error message and return? it;s more readable to me

- I would avoid relate specific scanners in this generic layer. it makes place for changes in case of future extensions

Reply:
- The exception won't get caught. in case of unsupported scanner the application will exit with a proper message.

- Regarding the message - I will drop the specific scanners

Reply:
Why do we want to crash the application in this case?
I think we should just write the proper message. 
think of a customer that doesn't want to have global scanner for all his projects, he might have some attempts without real scanner, but he can add them to his config as code later